### PR TITLE
Update wiznote to 2.6.0

### PR DESCRIPTION
**Pull request by vitahlin**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/wiznote.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'wiznote' do
-  version '2.5.8,2018-01-12'
-  sha256 '5428cf52af8544c741009d9b56fc0264b53aa4bd7b6ee8b8a08160fa9a546d3b'
+  version '2.6.0,2018-06-15'
+  sha256 '4ed19bf7cf1c69dcf648aa82a0432b18a77693a13b9b6dd832300d76fa38000b'
 
   url "https://get.wiz.cn/wiznote-macos-#{version.after_comma}.dmg"
```
Comment (vitorgalvao) [inline Casks/wiznote.rb]: What matters is the final download URL after redirects, not the one they show on the page.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/wiznote.rb (hunk 1 above)
Comment (vitahlin) [reply Casks/wiznote.rb]: They have changed the download URL to `http://url.wiz.cn/u/mac` after version 2.6.

Comment (vitorgalvao) [reply Casks/wiznote.rb]: > They have changed the download URL to `http://url.wiz.cn/u/mac` after version 2.6.

No, they have not. Like I said, that is the URL they show on the page, not the final one after redirection. Go to that URL on your browser and then check your browser’s `downloads` page to see it.

Comment (vitahlin) [reply Casks/wiznote.rb]: I got that.
The download URL do not need to update.
Thanks.

**Pull request merged**